requirements_dev.txt

```diff
@@ -0,0 +1,67 @@
+aenum==2.1.2
```

Comment:
sorry, i did not mean to create a `requirements_dev.txt` file. or if you'd do it, then create an empty one with only your new test-related dependencies but not all normal requirements. let's chat in the neopython channel on discord about this.

Reply:
okay, I am deleting it